packages/items_repository/lib/src/data/models/item_model.dart

```diff
@@ -34,41 +35,68 @@ abstract class ItemModel implements ItemEntity {
   final StorageEntity storage;
 
   @override
-  final Measure remainingMeasure;
+  final int? unsealedLifeTimeInDays;
+
+  @override
+  final int amount;
 
   ItemModel copyWith({
-    DateTime? expirationDate,
-    Measure? remainingMeasure,
+    DateTime? initialExpiryDate,
     StorageEntity? storage,
+    Optional<DateTime?> openedAt,
+    int? amount,
+    int? unsealedLifeTimeInDays,
   });
 
   @override
   ItemStatus get status {
```

Comment:
non mi piace che questo "itemStatus" sia nel model. Non fa parte del dominio. Ci vedo più giusto un getter `bool get isExpired` etc. In particolare perché gli stati si overlappano. Preferisco `if (item.isExpired)` piuttosto che `if (status == ItemStatus.expired)`. Se è comodo nella gestione/serializzazione dei path degli screen allora è chiaro che è una cosa UI quindi `ItemStatus` può essere definito lato view e fine

Reply:
lo mettiamo in qualche altro file non nei model

Reply:
> Però usa dei dati del modello, purtroppo non può essere definito nell'item perché è un'interfaccia

Quello che intendo è che non è inerente al dominio

Reply:
Spostato all'interno della UI.